jekyll/_cci2/caching.md

```diff
@@ -197,7 +197,9 @@ Caches created via the `save_cache` step are stored for up to 15 days.
 ### Clearing cache
 {: #clearing-cache }
 
-If you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
+Caches currently cannot be cleared; however, if you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
```

Comment:
I would potentially reword this to: 
" Caches currently cannot be cleared. If you need to generate a new set of caches because you've updated language or build management tool versions, simply update the cache key, similar to the previous example. 

Updating the cache key on save and restore steps in the `config.yml` file will then generate new sets of caches from that point onwards. Please note that older commits using the previous keys may still generate & save cache for builds, so remember to rebase after the `config.yml` changes when possible."